src/viperleed/__init__.py

```diff
@@ -11,7 +11,7 @@
 __copyright__ = 'Copyright (c) 2019-2024 ViPErLEED developers'
 __created__ = '2020-08-03'
 __license__ = 'GPLv3+'
-__version__ = '0.12.2'
+__version__ = '0.12.3'
```

Comment:
Please undo this one for merging into 0.12.2

Reply:
Ah, good catch!